Source/PlasticSourceControl/Private/PlasticSourceControlOperations.cpp

```diff
@@ -552,18 +552,19 @@ bool FPlasticRevertWorker::Execute(FPlasticSourceControlCommand& InCommand)
 
 	check(InCommand.Operation->GetName() == GetName());
 
-	TArray<FString> Files;
+	TArray<FString> Files = InCommand.Files;
```

Comment:
I assume that you changed that for extra readability? (might be just some leftover of more changes you made earlier?)
In this case we should align and change it everywhere; it came from the idea to avoid any unneeded copy (which is probably very debatable in this context). I don't care too much in what we decide, but I do think it's worth keeping coherence.

Reply:
This small refactor seeks both readability and combination of two sequential `#if ENGINE_MAJOR_VERSION == 5` directives. We took advantage of it to get rid of that ugly `#endif` in the middle of an `if` statement.

```
#if ENGINE_MAJOR_VERSION == 5
    if (...)
    {
        ...
    }
#endif
    {
        Files = InCommand.Files;
    }
```